action/template.go

```diff
@@ -28,8 +29,12 @@ func GenerateTemplate(out io.Writer, in io.Reader, vals interface{}) {
 }
 
 // Template renders a template to an output file.
-func Template(out, in, data string) {
+func Template(out, in, data string, force bool) error {
 	var dest io.Writer
+	_, err := os.Stat(out)
+	if !force && err == nil {
+		return errors.New("string to overwrite an exiting file please re-run the command with -f flag")
```

Comment:
it would be helpful (not blocking for this PR) to put this error into a global, non-exported `var`, so you don't have to allocate one on each call that fails